robotd/master.py

```diff
@@ -11,12 +11,23 @@
 import pyudev
 import setproctitle
 
-from robotd.devices import BOARDS
+import textwrap
+
+from robotd.devices import BOARDS, STARTUP_BOARDS
+
+
+def _send(connection, message):
+    # Split the message into buf_size chunks
+    buf_size = BoardRunner.SOCK_BUFFER_SIZE
+    for msg in [message[i:i + buf_size] for i in range(0, len(message), buf_size)]:
+        connection.send(msg)
```

Comment:
The issue is the vision code produces Large packets, I wasn't happy with receiving 524288 byte packets just to make sure it can handle the worst possible case of the camera seeing all 512 markers. Plus this number would need to be arbitrarily raised if we added more info to the vision API. It felt better to have a dynamic packet size